managed/devops/bin/yb_platform_util.py

```diff
@@ -0,0 +1,698 @@
+        universe_json = self.__call_api(universe_delete_url, is_delete=True)
+        task_id = universe_json['taskUUID']
+
+        print('Universe delete requested successfully. Use {0} as task id to get status of universe.'.format(task_id))
+        if not self.args.no_wait:
+            self.__task_progress_bar(task_id)
+
+    def __validate_universe_name(self, universe_name):
+        """
+        Get the universe details and store it in a json file after formatting the json.
+
+        :param universe_name: Name of the universe to take a json data from.
+        :return: None
+        """
+        # TODO need to update API URL once PR:- https://github.com/yugabyte/yugabyte-db/pull/8367 get merged.
+        universe_delete_url = '{0}/api/v1/customers/{1}/universes/find/{2}'.format(
+            self.base_url, self.customer_uuid, universe_name)
+
+        self.__call_api(universe_delete_url)
+        
+
+    def create_universe(self):
+        """
+        Create the universe using the json and provided universe name.
+
+        :return: None
+        """
+        if not self.args.file:
+            sys.stderr.write("Input json file required. Use '-f|--file <file_path>' to pass json input file.")
+            exit(1)
+        input_file = script_path + '/' + self.args.file
+        universe_name = self.args.universe_name
+
+        configure_json, universe_name = self.__modify_universe_config(input_file, universe_name)
+
+        self.__validate_universe_name(universe_name)
+
+        universe_config_json = self.__post_universe_config(configure_json)
+        # add the missing fields in the config json.
+        universe_config_json['clusterOperation'] = 'CREATE'
+        universe_config_json['currentClusterType'] = 'PRIMARY'
+        # Call universe config api to get the update config data.
+        universe_config_json = self.__post_universe_config(universe_config_json)
+        if universe_config_json:
+            task_id = self.__create_universe_from_config(universe_config_json)
+            print('Universe create requested successfully. Use {0} as task id to get status of universe.'.format(task_id))
+            if not self.args.no_wait:
+                self.__task_progress_bar(task_id)
+        else:
+            sys.stderr.write('Unable to Create Universe for Customer {0}'.format(customer_uuid))
+
+
+    def get_single_customer_uuid(self):
+        """
+        Get customer UUID. Raise error in case multiple customer is present. 
+
+        :return: None
+        """
+        if self.args.customer_uuid:
+            self.customer_uuid = self.args.customer_uuid
+        else:
+            customer_url = '{0}/api/v1/customers'.format(self.base_url)
+            data = self.__call_api(customer_url)
+            if (len(data) == 1):
+                self.customer_uuid = str(data[0])
+            else:
+                sys.stderr.write('Muliple customer UUID present, Please provide customer UUID')
```

Comment:
Can we list them here ?

Reply:
That can be an issue. Because one customer can see another customer's key and that is a security bridge. I had planned the same earlier than i thought about the security bridge. Later I discuss it with Chirag and Sridhar and removed it.